src/main/groovy/com/ullink/gradle/sonar/SonarScannerMsbuildExtension.groovy

```diff
@@ -0,0 +1,42 @@
+/*************************************************************************
+ * ULLINK CONFIDENTIAL INFORMATION
+ * _______________________________
+ *
+ * All Rights Reserved.
+ *
+ * NOTICE: This file and its content are the property of Ullink. The
+ * information included has been classified as Confidential and may
+ * not be copied, modified, distributed, or otherwise disseminated, in
+ * whole or part, without the express written permission of Ullink.
+ ************************************************************************/
+package com.ullink.gradle.sonar
+
+class SonarScannerMsbuildExtension {
```

Comment:
Very minor but I'd name this MSBuild, not Msbuild

Reply:
The classes in [gradle-msbuild-plugin](https://github.com/Ullink/gradle-msbuild-plugin) are named using the current way. I would like to stick to the convention.